calico-enterprise/multicluster/set-up-multi-cluster-management/helm-install/create-a-managed-cluster-helm.mdx

```diff
@@ -64,15 +64,17 @@ If you are installing on a cluster installed by EKS, GKE, AKS or Mirantis Kubern
   EOF
   ```
 
-1. Add any other customizations you require to `values.yaml`. To see values that can be customized in the chart, see the [helm docs](https://helm.sh/docs/) or run the following command:
+1. Add any other customizations you require to `values.yaml`. To see values that can be customized in the chart run the following command:
 
-  ```
-  helm show values projectcalico/tigera-operator --version tigera-operator-{{version}}
-  ```
+  <CodeBlock language='bash'>
+     {'{{version}}' === 'master'
+       ? `helm show values tigera/tigera-operator --version v0.0`
+       : `helm show values ./tigera-operator-{{chart_version_name}}.tgz`}
+  </CodeBlock>
 
 ### Install {{prodname}}
 
-To install a {{prodname}} [managed](../create-a-managed-cluster#value) cluster with Helm:
+To install a {{prodname}} [managed](create-a-managed-cluster-helm#value) cluster with Helm:
```

Comment:
This might be better served as a link to the "Value" section in the standard install page. The same comment applies for the management cluster page (and for the unversioned replicas of these pages)